### PR TITLE
CSHARP-4825: Make ReadConcernMajorityNotAvailableYet a retryable read error.

### PR DESCRIPTION
**Pull request by rstam**

### CODE REVIEW

File: specifications/retryable-reads/tests/README.rst [hunk 1]
```diff
@@ -9,8 +9,10 @@ Retryable Reads Tests
 Introduction
 ============
 
-The YAML and JSON files in this directory tree are platform-independent tests
-that drivers can use to prove their conformance to the Retryable Reads spec.
+The YAML and JSON files in the ``legacy`` and ``unified`` sub-directories are platform-independent tests
```
Comment (rstam) [inline specifications/retryable-reads/tests/README.rst:12]: I wonder if this file should just be deleted?

Comment (adelinowona) [reply specifications/retryable-reads/tests/README.rst:12]: Yeah I think we can just delete this. No need for us to even have any READMEs from the spec repo in our driver I think.

**Review by adelinowona — APPROVED** [review]
LGTM + remove the README

File: specifications/retryable-reads/tests/README.rst (hunk 1 above)
Comment (JamesKovacs) [reply specifications/retryable-reads/tests/README.rst:12]: It is in the spec repo. Best to leave this here for easier synchronization.

**Review by JamesKovacs — APPROVED** [review]
LGTM

**Pull request closed without merging**